gufe/tests/test_chemicalstate.py

```diff
@@ -0,0 +1,51 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+
+import pytest
+from openff.toolkit.topology import Molecule
+
+import gufe
+
+
+@pytest.fixture
+def prot_comp(PDB_181L_path):
+    yield gufe.ProteinComponent.from_pdbfile(PDB_181L_path)
+
+
+@pytest.fixture
+def solv_comp():
+    yield gufe.SolventComponent(ions=('K', 'Cl'))
```

Comment:
Some joke about chemical engineers preferring KCl over NaCl, but actually just to point out - NaCl might be a more common use case.